src/Renderer/PointsMaterial.js

```diff
@@ -69,8 +70,12 @@ class PointsMaterial extends THREE.RawShaderMaterial {
     constructor(options = {}) {
         const intensityRange = options.intensityRange || new THREE.Vector2(0, 1);
         const oiMaterial = options.orientedImageMaterial;
+        const classification = options.classification || ClassificationScheme.DEFAULT;
```

Comment:
For my own culture : what is the interest in creating a const for `classification` instead of setting `this.classification` value from `options.classification` directly ?

Reply:
it's spesific case with the extending `THREE.RawShaderMaterial` because its options constructor  don't support a unknown parameter. It's the raison we delete unknown parameters before constructor. 
After constructor, we set `this.classification`.